src/server/game/Entities/GameObject/GameObjectData.h

```diff
@@ -122,9 +122,9 @@ struct GameObjectTemplate
         {
             uint32 lockId;                                  //0 -> Lock.dbc
             uint32 level;                                   //1
-            uint32 diameter;                                //2 diameter for trap activation
+            uint32 radius;                                  //2 radius for trap activation
             uint32 spellId;                                 //3
-            uint32 type;                                    //4 0 trap with no despawn after cast. 1 trap despawns after cast. 2 bomb casts on spawn.
+            uint32 charges;                                 //4 0 trap with no despawn after cast. 1 trap despawns after cast. 2 trap despawns after cooldown after cast.
```

Comment:
Make up your mind here... you renamed it to `charges` and also at the same time adjusted the comment as if it was `type`

Reply:
Well.. I don't know to be honest. Now it will check increase the use counter then check if its equal to the charges value, if so, despawn. To have that behavior of the spirit alarms (despawning after 5 seconds / cooldown).